### ISSUE TITLE
README issue

### ISSUE THREAD

**Issue by MrMasrozYTLIVE** [TimeConqueror/Time-Core#7]
Hello,
All of the README files in this project and it's branches have one same issue:
`JSON-Based Bedrock Entity Animations - Easy-to-use system that allows you to play complex entity animations, which are parsed from JSON. More info here:(click)`
It should have clickable link at the end, but in all the branches that have README (about 80% of branches) have same README, without clickable `(click)`.

-----------
Мой Английский так плох....
TL;DR В ветках этого репозитория в описании нет кликабельной ссылки с информацией про анимации
